java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Order/EntreeOrdered.java

```diff
@@ -0,0 +1,14 @@
+package com.my.FoodTruckApp2.Order;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@Data
+@NoArgsConstructor
+public class EntreeOrdered {
+    private Integer orderId;
+    private Integer entreeId;
+}
```

Comment:
this doesn't seem to be getting used at all right now.

Reply:
i will delete it